src/style.css

```
@@ -0,0 +1,30 @@
+:root {
+  --height-box: 16px;
+  --width-box: 20px;
```

```suggestion
  --width-box: 16px;
```
and add margins to make it work like this